tests/trace/test_scored_calls_query.py

```diff
@@ -126,6 +126,7 @@ def predict(x):
     calls_high_level = client.get_calls(scored_by=[s0_name])
     assert call_ids(calls_low_level) == expected_ids
     assert call_ids(calls_high_level) == expected_ids
+    assert len(calls_high_level) == len(calls_low_level)
```

Comment:
nit: we are relying on an implementation detail: that get_calls returns a CallsIter that internally uses the stats endpoint to determine length. Let's take a world were we adding an optimization where after iterating through the list we update an internal size param so we can skip the stats call. We could do that and this test would not know any better. But as a result, it is no longer testing the functionality you desire. I'd recommend just explicitly calling the stats endpoint and checking that

Reply:
yup adding